src/protocols/LoRaWAN/LoRaWAN.cpp

```diff
@@ -716,56 +904,86 @@ int16_t LoRaWANNode::uplink(uint8_t* data, size_t len, uint8_t port, bool isConf
 
   // check maximum payload len as defined in phy
   if(len > this->band->payloadLenMax[this->dataRates[RADIOLIB_LORAWAN_CHANNEL_DIR_UPLINK]]) {
-    return(RADIOLIB_ERR_PACKET_TOO_LONG);
+    len = this->band->payloadLenMax[this->dataRates[RADIOLIB_LORAWAN_CHANNEL_DIR_UPLINK]];
+    // return(RADIOLIB_ERR_PACKET_TOO_LONG);
```

Comment:
Actually I am not completely sure about this one - the Specification Verification protocol TS008 requests a device to truncate payload. We could throw the error by default and then manually edit that line in case of running the verification protocol..

Reply:
I see - in that case pelase add comment referencing TS008 so that we know why is that commented-out line in there.